fink_filters/filter_anomaly_notification/filter_utils.py

```diff
@@ -0,0 +1,49 @@
+import time
+import requests
+from slackclient import SlackClient
+import tokens
+
+
+def msg_handler_slack(slack_data, channel_name, med):
+    slack_client = SlackClient(tokens.slack_token)
+    try:
+        channels = slack_client.api_call("conversations.list")['channels']
+        for channel in channels:
+            if channel['name'] == channel_name:
+                channel_buf = channel['id']
+                break
+    except KeyError:
+        requests.post("https://api.telegram.org/bot" + "/sendMessage", data={
+            "chat_id": "@fink_test",
+            "text": 'Slack API error'
+        }, timeout=8)
+        channel_buf = None
+    slack_data = [f'Median anomaly score overnight: {med}'] + slack_data
+    for slack_obj in slack_data:
+        if channel_buf:
+            slack_client.api_call(
+                "chat.postMessage",
+                channel=channel_buf,
+                text=slack_obj,
+                username='fink-bot'
+            )
+        time.sleep(3)
+
+def msg_handler_tg(tg_data, channel_id, med):
```

Comment:
Could you add a docstring to this function?

Reply:
Done!